packages/editor/.prettierignore

```diff
@@ -1,3 +1,4 @@
 # this also checks .gitignore
 
-dist
\ No newline at end of file
+dist
+CHANGELOG.md
```

Comment:
Instead of gitignoring it, what about adding a prettier step `yarn prettier -w packages/editor/CHANGELOG.md` before commiting it?